systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -0,0 +1,291 @@
+        KubeResourceManager.get().deleteResource(getBundleRoleBindings());
+        KubeResourceManager.get().deleteResource(getBundleClusterRoleBindings());
+        KubeResourceManager.get().deleteResource(getBundleConfigMap());
+        KubeResourceManager.get().deleteResource(getBundleDeployment());
+    }
+
+    private ClusterRoleBinding[] getStrimziWatchAllCrbs() {
+        return new ClusterRoleBinding[]{
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME + "-namespaced")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME + "-namespaced")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build(),
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME  + "-entity-operator")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME  + "-entity-operator")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build(),
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME + "-watched")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME + "-watched")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build()
+        };
+    }
+
+    private ConfigMap getBundleConfigMap() {
+        return new ConfigMapBuilder(TestFrameUtils.configFromYaml(strimziResources.stream()
+            .filter(file -> file.getName().matches(".*050-ConfigMap-.*\\.yaml")).toList().get(0), ConfigMap.class))
```

Comment:
I know it seems silly, but it might be good to change the `matches` to `startsWith` and drop the `yaml` part since these regex were all flagged as a security hotspot.
```suggestion
            .filter(file -> file.getName().startsWith("050-ConfigMap-")).toList().get(0), ConfigMap.class))
```